lldb/source/Commands/CommandObjectExpression.cpp

```diff
@@ -148,6 +148,10 @@ Status CommandObjectExpression::CommandOptions::SetOptionValue(
     top_level = true;
     break;
 
+  case 's':
+    evaluate_self_as_generic = true;
```

Comment:
This is really specific. What would you think about making a parameter that controls all generic type parameter binding?

Reply:
@adrian-prantl what do you think of `unbound-generic-params`? `dont-bind-generic-params`?